library/jetty

```diff
@@ -1,20 +1,21 @@
 # maintainer: Mike Dillon <mike@appropriate.io> (@md5)
+# maintainer: Greg Wilkins <gregw@webtide.com> (@gregw)
```

Comment:
@gregw Should this have your info here or something more generic for the Jetty Project or Webtide?